src/vl-mapactions-draw-action.js

```diff
@@ -32,8 +32,20 @@ export class VlDrawAction extends VlMapAction {
     });
     const drawInteraction = new Draw(options);
     interactions.push(drawInteraction);
-    if (options.snapping) {
-      interactions.push(new VlSnapInteraction(options.snapping.layer || layer));
+    if (options.snapping !== undefined) {
+      if (typeof options.snapping === 'boolean') {
+        if (options.snapping) {
+          interactions.push(new VlSnapInteraction({source: layer.getSource()}));
+        }
+      } else {
+        const snappingOptions = Object.assign({}, options.snapping);
```

Comment:
Waarom is hier een `Object.assign` nodig?

Reply:
Ik vond het properder om niet de meegegeven properties te gaan manipuleren en die gewoon als defaults te gebruiken ...